Sources/Compiler/IR/InstructionID.swift

```diff
@@ -1,30 +1,30 @@
-/// A stable identity denoting an instruction in a module.
+/// The stable identity of an instruction in its module.
 ///
 /// - SeeAlso: `InstructionIndex`
 public struct InstructionID: Hashable {
 
-  /// The index of the function in which the instruction resides.
+  /// The function containing this identity.
```

Comment:
```suggestion
  /// The function containing the instruction.
```

or if you must,

```suggestion
  /// The function containing the identified instruction.
```

What you wrote here suggests that you can query the function and find this identity stored therein.